### ISSUE TITLE
Clean up the Network

### ISSUE THREAD

**Issue by ma11hew28** [schacon/gitbook#15]
Hey Scott,

It'd be nice if you or someone took a weekend to clean up the Git Network. I was going to fork this repo, but I like to look at the network tab (https://github.com/schacon/gitbook/network) before doing that... and didn't know which repo to fork. It's pretty hairy.

It'd be nice if you could merge all the other branches into your own and then make a commit (if necessary to make some other tweaks to that person's contributions) after each merge. That way, all the other branches disappear (because your branch now has all their commits), and it becomes more clear that I should just fork your repo.

The general strategy I've adopted is just to merge all pull-requests ASAP (even if there's things I would change about them and don't have time to make an extra "enhancement" commit after the merge) so that everyone's working on the same code base. Not sure if that's good practice... but yeah...

Matt

**Comment by schacon** [comment]
Yeah, the main issue is that I completely stopped working on or maintaining this years ago when I started on the Pro Git book.  There I have merged several hundred PRs probably and keep it pretty much up to date and am planning on doing more.  It's better laid out and planned and I think has more useful information in it and is easier to read.  I only barely have time to maintain one book, so that's the one I'm focusing on.  I'm trying to lead more people towards that book and deprecate this one.
